src/Stryker.CLI/Stryker.CLI/Logging/LoggingInitializer.cs

```diff
@@ -31,16 +31,25 @@ public void SetupLogOptions(IStrykerInputs inputs, IFileSystem fileSystem = null
             ApplicationLogging.ConfigureLogger(logLevel, logToFile, outputPath);
         }
 
-        private string CreateOutputPath(string basePath, IFileSystem fileSystem)
+        private string CreateOutputPath(IStrykerInputs inputs, IFileSystem fileSystem)
         {
-            var strykerDir = "StrykerOutput";
+            var basePath = inputs.OutputBasePathInput.SuppliedInput ?? Path.Combine(inputs.BasePathInput.SuppliedInput, "StrykerOutput");
+            string outputPath;
+
+            if (Path.IsPathRooted(basePath))
+            {
+                outputPath = Path.Combine(basePath, DateTime.Now.ToString("yyyy-MM-dd.HH-mm-ss"));
+            }
+            else
+            {
+                outputPath = Path.Combine(inputs.BasePathInput.SuppliedInput, basePath, DateTime.Now.ToString("yyyy-MM-dd.HH-mm-ss"));
+            }
```

Comment:
If the path is supplied by the user, we do not generate the name of the folder using the timestamp. We use the exact path as supplied by the user.